1337up23-web-owasp.md

```diff
@@ -0,0 +1,185 @@
+---
+title: "1337UP web/OWASP"
+publishDate: "20 Nov 2023"
+description: "Author: cpp.dog"
+tags: ["web", "1337up"]
+---
+
+#### Description
+
+Everyone knows the OWASP top 10 are vulnerable, I guess this might be too 💀
+
+Author: Ivars Vids
+
+#### Code inspection
+
+Sourceless.
+
+P.S.
+After CTF has ended, some participants shared the dirbusted source file.
+`https://owasp.ctf.intigriti.io/search.php.save`
+
+Please note that I didn't know about this file, because dirbusting is generally prohibited in most CTFs, unless it is explicitly stated in the challenge description. 
```

Comment:
```suggestion
Please note that we didn't know about this file, because dirbusting is generally prohibited in most CTFs, unless it is explicitly stated in the challenge description. 
```